internal/core/src/index/InvertedIndexTantivy.cpp

```diff
@@ -261,8 +261,11 @@ InvertedIndexTantivy<T>::Load(milvus::tracer::TraceContext ctx,
         inverted_index_files.end());
     disk_file_manager_->CacheIndexToDisk(inverted_index_files);
     path_ = prefix;
-    wrapper_ = std::make_shared<TantivyIndexWrapper>(prefix.c_str(),
-                                                     milvus::index::SetBitset);
+
+    auto load_in_mmap =
+        GetValueFromConfig<bool>(config, ENABLE_MMAP).value_or(true);
+    wrapper_ = std::make_shared<TantivyIndexWrapper>(
+        prefix.c_str(), load_in_mmap, milvus::index::SetBitset);
```

Comment:
is it true that if we choose to load in ram, we still delete the files when releasing? can we delete ealier?